### DISCUSSION TITLE
Survey for Quasar Scaffolding beta, out now!

### DISCUSSION THREAD

**Discussion by IlCallo** [quasarframework/quasar#10053] [Announcements]
Fill out our new survey and spread the word!
Help us decide what to include in Quasar Scaffolding beta version
https://bit.ly/2VtIGK9

![image](https://user-images.githubusercontent.com/10036108/125299085-162acc80-e329-11eb-9be0-bf5fc699b8af.png)

**Comment by JokerMartini** [comment]
What is Scaffolding?

**Reply by IlCallo** [reply]
It'll be a paid product similar to Laravel JetStream and Laravel Nova, where we'll offer pre-build templates and integrations

Into last community survey many devs pointed out they won't be able to support Quasar with donations, but they may be able to do so with a paid product/service.
This is a first step in that direction :)

**Reply by JokerMartini** [reply]
Hopefully that excells. I would love for quasar to continue to grow and be used. There are not many solid alternatives.

**Comment by IlCallo** [comment]
The survey is now closed, thanks to everyone who participated!
